.github/workflows/Stale.yml

```diff
@@ -0,0 +1,93 @@
+        required: false
+        type: number
+      days-before-pr-stale:
+        description: 'Override days-before-stale for PRs only'
+        default: 60
+        required: false
+        type: number
+      days-before-issue-close:
+        description: 'Idle number of days before closing stale issues'
+        default: 7
+        required: false
+        type: number
+      days-before-pr-close:
+        description: 'Idle number of days before closing stale PRs'
+        default: 7
+        required: false
+        type: number
+      stale-issue-message:
+        description: 'Comment made on stale issues'
+        default: >
+          This issue has been automatically marked as stale because it has not had
+          activity in 45 days. It will be closed if no further activity occurs within
+          7 days. Thank you for your contributions.
+        required: false
+        type: string
+      stale-pr-message:
+        description: 'Comment made on stale PRs'
+        default: >
+          This PR has been automatically marked as stale because it has not had
+          activity in 60 days. It will be closed if no further activity occurs within
+          7 days. Thank you for your contributions.
+        required: false
+        type: string
+      close-issue-message:
+        description: 'Comment made on stale issues when closed'
+        default: >
+          This issue has been automatically been closed because it did not have any
+          activity in 45 days and no follow up within 7 days after being marked stale.
+          Thank you for your contributions.
+        required: false
+        type: string
+      close-pr-message:
+        description: 'Comment made on stale PRs when closed'
+        default: >
+          This pull request has been automatically been closed because it did not have any
+          activity in 60 days and no follow up within 7 days after being marked stale.
+          Thank you for your contributions.
+        required: false
+        type: string
+
+jobs:
+  stale:
+    name: Stale
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+      pull-requests: write
+
+    steps:
+    - name: Check for Stale Items
+      uses: actions/stale@v6
+      with:
+        days-before-issue-stale: ${{ inputs.days-before-issue-stale }}
+        days-before-pr-stale: ${{ inputs.days-before-pr-stale }}
+        days-before-issue-close: ${{ inputs.days-before-issue-close }}
+        days-before-pr-close: ${{ inputs.days-before-pr-close }}
+        stale-issue-message: ${{ inputs.stale-issue-message }}
+        stale-pr-message: ${{ inputs.stale-pr-message }}
+        close-issue-message: ${{ inputs.close-issue-message }}
+        close-pr-message: ${{ inputs.close-pr-message }}
+        stale-issue-label: 'stale: issue'
+        stale-pr-label: 'stale: pr'
+        exempt-issue-labels: 'impact: security,state: backlog'
+        exempt-pr-labels: 'impact: security,state: backlog'

```

Comment:
sorry.  after looking at it I think there should be a couple of changes.  

1.  No spaces between ":" and sub section.  should be "impact:security".
2. I don't think we should use "stale:".  Docs we have that we are kind of following would have pattern "state:stale"
3. Is there any reason to have stale pr and stale issue independent?   Anything unique that you would trigger.
4. We also

Reply:
(1) will update to drop spaces
(2) will update to "state:stale"
(3) as far as the labels go, I think it's fine to combine
(4) let me know if this was a typo or something else expected

Reply:
All changes made in PR commit https://github.com/microsoft/mu_devops/pull/27/commits/8999366cccb9281c328b3dbb011bc74b7b3c2e0c